app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<network-security-config>
+    <base-config cleartextTrafficPermitted="true">
```

Comment:
Maybe this could be limited to hassio.local domain?

E.g. : 
```
    <domain-config cleartextTrafficPermitted="false">
        <domain includeSubdomains="true">hassio.local</domain>
    </domain-config>
```

Reply:
For #48 this is unrelated. Currently this is true (base-config) since http is allowed. But I think there are users that are not using mDNS and https, so they will not be satisfied by `hassio.local`.  Actually `hassio.local` will not add any security. This address may be spoofed. And anyway this is plain http - it may be sniffed. So, I don't think we should change this line till we support http.

Reply:
The cleartext may be needed to allow http communication between the app and the Home Assistant instance. There is no requirement for all of those instances to be httpS ones. At least, within a secure LAN. It's not the best security practice but may be quite overkill for a non WAN exposed Home Assistant.

Reply:
I'd leave this line in so those running Android 7.0 (API 24) and above get the same behavior that was introduced in e54311ec with the setting in AndroidManifest.xml. (The setting in AndroidManifest.xml is ignored in 7.0+ if there is a network security config.)